src/proj.h

```diff
@@ -328,6 +340,17 @@ extern const PJ      *proj_shutdown;
 #endif
 
 
+/* Set or read error level */
+int  proj_errno (PJ *P);
+void proj_errno_set (PJ *P, int err);
```

Comment:
Yeah, I have been thinking about setting up a project here on GitHub for the coming release. I'll try to play around with that today I think.